modules/stack/main.tf

```diff
@@ -26,7 +27,7 @@ resource "spacelift_mounted_file" "stack_config" {
     for k, v in var.component_vars : k => jsondecode(v)
   }))
 
-  write_only = true
+  write_only = false
```

Comment:
I'm setting this permanently as it doesn't contain anything sensitive and is frequently useful for troubleshooting stack config issues.